content/docs/reference/cel.md

```diff
@@ -9,7 +9,31 @@ CEL allows writing simple expressions based on the request context that evaluate
 
 ## Example expressions
 
-Below shows some example expressions that can be used in logs, traces, etc:
+Review the following examples of expressions for different uses cases.
+
+### Default function {#default-function}
+
+You can use the `default` function to provide a fallback value if the expression cannot be resolved.
+
+Request header fallback with example of an anonymous user:
+
+```
+default(request.headers["x-user-id"], "anonymous")
+```
+
+Nested object fallback with example of light theme:
+
+```
+default(request.body.user.preferences.theme, "light")
```

Comment:
request.body is a string so you cannot actually do this. It would need to be something like `json(request.body).user` to parse it as json